docs/releases/5.1.7.txt

```diff
@@ -9,4 +9,6 @@ Django 5.1.7 fixes several bugs in 5.1.6.
 Bugfixes
 ========
 
-* ...
+* Fixed a bug in Django 5.1.6 where the ``{% querystring %}`` template tag
```

Comment:
I think the regression is in `5.1` not in `5.1.6`? i.e. I don't think we landed a change for `5.1.6` that broke this, the logic that is being fixed was added along with the feature before alpha release.

Reply:
Good spot!